signac/contrib/project.py

```diff
@@ -766,15 +766,16 @@ def num_jobs(self):
             Count of initialized jobs.
 
         """
+        return len(self)
+
+    def __len__(self):
```

Comment:
Thanks for using a `def` here, that's clearer.

Reply:
Also the other method will be gone soon so there won't be anything to alias :)